starname/chain.json

```diff
@@ -7,6 +7,56 @@
     "chain_id": "iov-mainnet-ibc",
     "bech32_prefix": "star",
     "slip44": 234,
+    "genesis": {
+        "genesis_url": "https://gist.githubusercontent.com/davepuchyr/6bea7bf369064d118195e9b15ea08a0f/raw/genesis.json"
+    },
+    "codebase": {
+        "git_repo": "https://github.com/iov-one/starnamed",
+        "recommended_version": "v0.10.18",
+        "compatible_versions": [
+            "v0.10.17", "v0.10.18"
+        ]
+    },
+    "peers": {
+        "persistent_peers": [
```

Comment:
Do you by chance have a list of the providers of each of these?

Reply:
Don't have them for now. Will confirm from the Starname team and update.